Makefile

```diff
@@ -59,6 +59,7 @@ ENVSUBST_BIN := envsubst
 ENVSUBST := $(TOOLS_BIN_DIR)/$(ENVSUBST_BIN)-drone
 SETUP_ENVTEST = $(TOOLS_BIN_DIR)/setup-envtest
 GINKGO := "$(ROOT_DIR)/$(TOOLS_BIN_DIR)/ginkgo"
+GINKGO_BIN := ginkgo
```

Comment:
Maybe swap lines 61 and 62 and use `$(GINKGO_BIN)` when defining `GINKGO` ?

Reply:
Yap thats better . Good catch